From 91a0b75ae6a840ea8b6c9a629d118a9c992c383c Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Sat, 17 Feb 2024 20:15:27 +0000
Subject: [PATCH 3/3] appdata: Add new <developer> tag

Appstream 1.0 introduces a new <developer> tag which supersedes
<developer_name>. Add it, keeping <developer_name> for
backwards-compatibility.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-developer
---
 src/org.tuxpaint.Tuxpaint.appdata.xml.in | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/org.tuxpaint.Tuxpaint.appdata.xml.in b/src/org.tuxpaint.Tuxpaint.appdata.xml.in
index 799c1a16..9a46b23c 100644
--- a/src/org.tuxpaint.Tuxpaint.appdata.xml.in
+++ b/src/org.tuxpaint.Tuxpaint.appdata.xml.in
@@ -23,6 +23,9 @@
     </p>
   </description>
   <developer_name>Bill Kendrick, et al.</developer_name>
+  <developer id="newbreedsoftware.com">
+    <name>Bill Kendrick, et al.</name>
+  </developer>
   <url type="homepage">https://tuxpaint.org/</url>
   <url type="bugtracker">https://sourceforge.net/p/tuxpaint/_list/tickets</url>
   <url type="faq">https://tuxpaint.org/docs/en/html/FAQ.html</url>
-- 
2.43.0

